langchain/src/output_parsers/expression_type_handlers/string_literal_handler.ts

```diff
@@ -0,0 +1,26 @@
+import type { ESTree } from "meriyah";
```

Comment:
Yeah I tried moving it to dev/optional but the test-exports would just not work for me since it keeps trying to import meriyah, even when I add the self_query to requiresOptionalDependency in create-entrypoints. I think it's because I have to import the parser.

Reply:
Yeah I tried moving it to dev/optional but the test-exports would just not work for me since it keeps trying to import meriyah, even when I add the self_query to requiresOptionalDependency in create-entrypoints. I think it's because I have to import the parser that uses meriyah.